src/test/java/edu/hm/hafner/metric/FractionValueTest.java

```diff
@@ -14,12 +16,75 @@ class FractionValueTest {
     @Test
     void shouldCreateDelta() {
         var fifty = new FractionValue(Metric.LINE, Fraction.getFraction(50, 1));
+        var fiftyAgain = new FractionValue(Metric.LINE, Fraction.getFraction(50, 1));
         var hundred = new FractionValue(Metric.LINE, Fraction.getFraction(100, 1));
 
         assertThat(fifty.isBelowThreshold(50.1)).isTrue();
         assertThat(fifty.isBelowThreshold(50)).isFalse();
 
         assertThat(fifty.add(fifty)).isEqualTo(hundred);
         assertThat(fifty.max(hundred)).isEqualTo(hundred);
+        assertThat(fifty.max(fiftyAgain) == fifty).isTrue();
```

Comment:
Fachlich gesehen, macht der == Vergleich hier keinen Sinn.

```suggestion
        assertThat(fifty.max(fiftyAgain)).isEqualTo(fifty);
```